Aptoide/aptoide-client-v8
# [MOB-382] task: disable new feature notification and dialog for dark-theme

**What does this PR do?**

   Disable dark theme new feature notification & dialog in home.
   Removed code related to the dark theme notification itself but maintaining the more abstract code related to a new generic feature that can be used in the future.
   
**Database changed?**

   No

**Where should the reviewer start?**

- [ ] DarkThemeNewFeatureManager.kt

**How should this be manually tested?**

  check both - if they are disabled:
 - 2 min notification that is shown if you don't see home
 - bottom sheet dialog in home

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-382](https://aptoide.atlassian.net/browse/MOB-382)

**Code Review Checklist**
